pkg/controller/migmigration/directimagemigration.go

```diff
@@ -41,6 +42,24 @@ func (t *Task) getDirectImageMigration() (*migapi.DirectImageMigration, error) {
 	return nil, nil
 }
 
+func (t *Task) getDirectImageStreamMigrationForDIM(dim *migapi.DirectImageMigration) (*migapi.DirectImageStreamMigration, error) {
+
+	dimLabels := dim.GetCorrelationLabels()
+	selector := labels.SelectorFromSet(dimLabels)
+	dismList := migapi.DirectImageStreamMigrationList{}
+	err := t.Client.List(context.TODO(),
+		&k8sclient.ListOptions{
+			LabelSelector: selector,
+		},
+		&dismList)
+
+	if err != nil {
+		return nil, liberr.Wrap(err)
+	}
+
+	return &dismList.Items[0], nil
```

Comment:
can't Items be an empty list?

Reply:
Thanks for the catch @shawn-hurley , updated the code.